README.md

```diff
@@ -89,10 +91,8 @@ After you’ve tested your changes:
 
 Return to the Kwil docs home page in GitHub and navigate to [Pull Requests](https://github.com/kwilteam/docs/pulls).
 
-Click the Compare and pull request button, add a brief description of the changes made, including a motivation for why this will benefit the Kwil community, and then finally click the Create pull request button. 
+Click the Compare and pull request button, add a brief description of the changes made, including a motivation for why this will benefit the Kwil community, and then finally click the Create pull request button.
```

Comment:
I don't see a "Compare and pull request" button anymore.  
![image](https://github.com/kwilteam/docs/assets/140431406/d19b67cc-8e6d-4c7b-82e5-f81e225f2420)
Will reword this.